deltachat-ffi/src/providers.rs

```diff
@@ -0,0 +1,93 @@
+use deltachat::dc_tools::{as_str, dc_strdup, StrExt};
+extern crate deltachat_provider_overview;
+use deltachat_provider_overview::StatusState;
+use std::ptr;
+
+#[no_mangle]
+pub type dc_provider = deltachat_provider_overview::Provider;
```

Comment:
`dc_provider_t` is the ffi convention for these.

Reply:
Done.